### DISCUSSION TITLE
math: inline math can't be surrounded by brackets, quotation marks etc.

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#30157] [Other Feature Feedback, Questions, & Ideas]
When putting any character around inline math, e.g.,
```
$\pi$, "$\pi$", ($\pi$)
```
it won't work. The above:

- $\pi$, "$\pi$", ($\pi$)

Screenshot:

![screenshot](https://user-images.githubusercontent.com/181628/185718204-8916b249-dd2a-490f-86e0-4b909afea07f.png)

Note how awkwardly small the $\pi$ is, too. (This has been discussed [before](https://github.com/community/community/discussions/20936).)

**Comment by airtower-luna** [comment]
I just experimented a little, and it seems the problem is any non-whitespace character *in front of* the formula. After is fine. `"$\pi$` breaks, `$\pi$"` works. I assume the formatting code should check for word boundary instead of whitespace.

**Comment by nschloe** [comment]
One can now use inline math with backticks to solve the problem, "$`\pi`$".
